src/sic/sim/MainView.java

```diff
@@ -120,6 +120,7 @@ public void run() {
 
     public void updateView() {
         cpuView.updateView();
+        disassemblyView.clearLabelMap();
```

Comment:
It would be better to do this in the Clear memory / Clear all actions, since updateView is also called in many other cases.